### DISCUSSION TITLE
free referans with vpn (beleş referans)

### DISCUSSION THREAD

**Discussion by Georgicusa** [IBAX-io/go-ibax#1150] [Ideas]
### free referans with vpn (beleş referans)

People can easily get references using VPN. He wants a twitter account of at least 3 months, but there are a lot of twitter accounts that have been used for 2 years on the internet and people can buy them very cheaply. In this case, a fair distribution cannot be achieved. My suggestion would be at least some motion video verification if not kyc. (İnsanlar VPN kullanarak kolayca referans alabilirler. En az 3 aylık twitter hesabı istiyor ama internette 2 yıllık bir sürü twitter hesabı var ve insanlar çok ucuza satın alabiliyor. bu durumda ise adil bir dağıtım gerçekleşmemiş oluyor benim önerim en azından kyc olmasa bile bir hareketli video doğrulaması olabilir.)

### Suggested solution

yukarıda yazdım

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by ibaxio** [comment]
thanks for your advice.
